src/js/profile/wearable/widget/wearable/SnapListview.js

```diff
@@ -343,14 +343,11 @@
 			prototype._listItemAnimate = function (scrollValue) {
 				var self = this,
 					anim = self.options.animate,
-					animateCallback = self._callbacks[anim],
-					scrollPosition;
+					animateCallback = self._callbacks[anim];
 
 				if (animateCallback) {
-					scrollPosition = scrollValue || scrolling.getScrollPosition();
-
 					utilArray.forEach(self._listItems, function (item) {
-						item.animate(scrollPosition, animateCallback);
```

Comment:
this function is also called in other place
function scrollHandler(self, event) {
self._listItemAnimate(event && event.detail && event.detail.scrollTop);
we need to make sure that event will always exist in this function otherwise we will call this function with undefined scrollValue

Reply:
I didn't meer such scenario but ok we can prevent from such case.